documentation/modules/proc-installing-using-olm.adoc

```diff
@@ -65,7 +65,7 @@ metadata:
 spec:
   channel: alpha
   name: {OperatorName}
-  source: {OperatorSourceName}
+  source: {OperatorSourceConfigName}
```

Comment:
To me the change looks fine. However I think that for the `name` attribute in the other CR instances (`CatalogSourceConfig` and `OperatorSource), we should use the parameter as well.

And the more I look at this, the less I think this will work downstream. At least the publisher and display names should be replaced as well. But more important, the app registry will not contain downstream images. And the operator package name will be different as well.

For this PR I think it is fine, but I think we need some more work in general.

Reply:
@ctron ; is this OK to be merged then? PR says you've requested changes but at same time say its fine.

Reply:
I said it is fine in general, but taking a look at the `name` parameters, don't you think they should be changed as well?

Reply:
rebased and updated name params to use attributes.